src/ic-cdk/src/api/management_canister/http_request/types.rs

```diff
@@ -1,28 +1,11 @@
+#![allow(missing_docs)]
```

Comment:
Just to double-check, is this intentional?

Reply:
There is a technical difficulty to add doc comment here: https://github.com/dfinity/cdk-rs/pull/411/files#diff-17f4aa023e6fba8b904d3e7081f39a97edc98f50116d0db50bc027175541e256R8

It's a public struct that is defined by a macro. Adding doc comment directly above the macro doesn't work. Not sure where to put it.

Reply:
I found a solution which requires more recent Rust toolchain.

I added a TODO in comment for upgrading Rust.

The Jira ticket is here: https://dfinity.atlassian.net/browse/SDK-1183